PWGHF/Tasks/taskD0parametrizedPID.cxx

```diff
@@ -62,9 +62,9 @@ struct TaskD0parametrizedPIDMC {
   //Configurable<double> centralitySelectionMax{"centralitySelectionMax", 30000.0, "Higher boundary of centrality selection"};
 
   Filter filterSelectCandidates = (aod::hf_selcandidate_d0_parametrizedPID::isSelD0NoPID >= 1 || aod::hf_selcandidate_d0_parametrizedPID::isSelD0barNoPID >= 1);
-  using McParticlesHf = soa::Join<aod::McParticles_000, aod::HfCandProng2MCGen>;
-  void process(soa::Filtered<soa::Join<aod::HfCandProng2, aod::HFSelD0CandidateparametrizedPID, aod::HfCandProng2MCRec>> const& candidates, soa::Join<aod::McParticles_000, aod::HfCandProng2MCGen> const& particlesMC, aod::BigTracksMC const& tracks)
-  // void process(const o2::aod::Collision& collision, soa::Filtered<soa::Join<aod::HfCandProng2, aod::HFSelD0CandidateparametrizedPID, aod::HfCandProng2MCRec>> const& candidates, soa::Join<aod::McParticles_000, aod::HfCandProng2MCGen> const& particlesMC, aod::BigTracksMC const& tracks)
+  using McParticlesHf = soa::Join<aod::McParticles, aod::HfCandProng2MCGen>;
+  void process(soa::Filtered<soa::Join<aod::HfCandProng2, aod::HFSelD0CandidateparametrizedPID, aod::HfCandProng2MCRec>> const& candidates, soa::Join<aod::McParticles, aod::HfCandProng2MCGen> const& particlesMC, aod::BigTracksMC const& tracks)
```

Comment:
Why not use the `using` in the argument list?

Reply:
This does not work, but I will update it with the method to retrieve the first/last daughters implemented in https://github.com/AliceO2Group/AliceO2/pull/8113